dapp/src/components/shared/TokenAmountForm/index.tsx

```diff
@@ -37,7 +37,7 @@ const TokenAmountForm = withFormik<TokenAmountFormProps, TokenAmountFormValues>(
       props.onSubmitForm(values)
     },
     validateOnBlur: false,
-    validateOnChange: false,
+    validateOnChange: true,
```

Comment:
We wanted to avoid running the validation on any input change for the amount field.
Currently, when you start entering the value starting with `0` you will immediately see the validation error. We want to run the validation when the user exits the input field.
So `validateOnBlur` seems like a better fit.

Reply:
Yeah, I agree. Let's change it.

Reply:
Ref commit: 878b1f43414632797ac00b8b3f7b6a405e6b1bb4